documentation/index.md

```diff
@@ -6,12 +6,13 @@ title: Documentation
 <div class="row">
   <div class="col-md-9">
     <div class="well">
-      <h4><a href="usecases">Use Cases</a></h4>
+      <h4><a href="/usecases">Use Cases</a></h4>
```

Comment:
for some reason it works even w/o the leading slash, good to make unambiguous tho